curriculum/challenges/english/08-coding-interview-prep/algorithms/implement-merge-sort.english.md

```diff
@@ -33,7 +33,7 @@ tests:
   - text: <code>mergeSort</code> should return an array that is unchanged except for order.
     testString: assert.sameMembers(mergeSort([1,4,2,8,345,123,43,32,5643,63,123,43,2,55,1,234,92]), [1,4,2,8,345,123,43,32,5643,63,123,43,2,55,1,234,92]);
   - text: <code>mergeSort</code> should not use the built-in <code>.sort()</code> method.
-    testString: assert(!code.match(/\.?[\s\S]*?sort/));
+    testString: assert(!removeJSComments(code).match(/\.?[\s\S]*?sort\(\)/));
```

Comment:
```suggestion
    testString: assert(!removeJSComments(code).match(/\.?[\s\S]*?sort\s*\(/));
```

I think this will be a little more robust. I used the sort method with arguments and it wasn't caught.
e.g. `.sort((a,b) => { ... })`

Reply:
@moT01 Nice catch